R/expr__string.R

```diff
@@ -862,29 +862,31 @@ ExprStr_slice = function(offset, length = NULL) {
 #' df$select(pl$col("a")$str$explode())
 ExprStr_explode = function() {
   .pr$Expr$str_explode(self) |>
-    unwrap("in str$explode():")
+    unwrap("in $str$explode():")
 }
 
-# TODO: rename to `to_integer`
-#' Parse integers with base radix from strings
+
+#' Convert a String column into an Int64 column with base radix
 #'
-#' @description Parse integers with base 2 by default.
-#' @keywords ExprStr
-#' @param base Positive integer which is the base of the string we are parsing.
-#' Default is 2.
-#' @param strict If `TRUE` (default), integer overflow will raise an error.
-#' Otherwise, they will be converted to `null`.
-#' @return Expr: Series of dtype i32.
+#' @param ... Ignored.
+#' @param base A positive integer or expression which is the base of the string
+#' we are parsing. Characters are parsed as column names. Default: `10L`.
+#' @param strict A logical. If `TRUE` (default), raise any ParseError or overflow
+#' as ComputeError. If `FALSE`, silently convert to `null`.
```

Comment:
```suggestion
#' @param strict If `TRUE` (default), parsing errors or integer overflow will
#' raise an error. Otherwise, they will be converted to `null`.
```

Reply:
I think it is actually only for `FALSE`, not otherwise. (This is a subtle nuance difference, such as when strictly logical may take `NA`, as opposed to when `isTRUE()` is used to determine it.......)

Reply:
In addition, this is copied from the Python documentation.

Reply:
Ok to replace "otherwise", but I think we should remove the ComputeError and ParseError since they don't mean much to R users